frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,16 +6,30 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { isApplicationSelected } from '@/store/ApplicationState';
+import { isApplicationSelected,selectedApplicationShortDisplayText } from '@/store/ApplicationState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
+const initialLabel = () => {
```

Comment:
Could you put a "comment" on why it is called "initial"?
Is it only for initial page load?
It is not clear to me since you are watching for something change and executing this function again.
If it is not only "initial", may be better adjust the function naming.

Reply:
I can do that